resources/views/components/input/range-slider.blade.php

```diff
@@ -0,0 +1,36 @@
+<range-slider v-slot="rangeInputScope" {{ $attributes }}>
+    <div class="flex relative h-16 mx-5">
+        <input
+            type="range"
+            v-bind:disabled="!canRefine"
+            v-bind:min="rangeInputScope.range.min"
+            v-bind:max="rangeInputScope.range.max"
+            v-model="rangeInputScope.minValue"
+            v-on:change="rangeInputScope.updateRefinement"
+            class="absolute pointer-events-none appearance-none z-20 h-5 w-full opacity-0 cursor-pointer [&::-moz-range-thumb]:pointer-events-auto [&::-webkit-slider-thumb]:pointer-events-auto"
+        >
+
+        <input
+            type="range"
+            v-bind:disabled="!canRefine"
+            v-bind:min="rangeInputScope.range.min"
+            v-bind:max="rangeInputScope.range.max"
+            v-model="rangeInputScope.maxValue"
+            v-on:change="rangeInputScope.updateRefinement"
+            class="absolute pointer-events-none appearance-none z-20 h-5 w-full opacity-0 cursor-pointer [&::-moz-range-thumb]:pointer-events-auto [&::-webkit-slider-thumb]:pointer-events-auto"
+        >
```

Comment:
Maybe we could also add number inputs in combination, it's something many webshops have and want.

Reply:
Definitely! I figured I would just rebuild the functionality we had before, but that would be fairly simple to add.